src/stats/probs/cdf.rs

```diff
@@ -289,11 +305,15 @@ impl<T: Ord> CDF<T> {
     }
 
     /// Return total probability of the `CDF`.
+    ///
+    /// Time complexity: O(n), where n is the number of `Entry`s in `CDF`.
```

Comment:
```suggestion
```
This is O(1), since `self.inner` is a `Vec`.

Reply:
Ah, I misinterpreted the `map_or()` as something of an iterator, but it's a function on the `Option<>` that `.last()` returns. But shouldn't we then state the `O(1)` instead of skipping runtime complexity altogether?

Reply:
Yes, perhaps!

Reply:
OK, done... :D